pyScienceMode2/sciencemode.py

```diff
@@ -270,6 +278,9 @@ def _thread_catch_ack(self):
         """
         Compare the command sent and received by the rehastim and retrieve the data sent by the motomed if motomed flag is true.
         """
+        from pyScienceMode2 import Rehastim2Commands
```

Comment:
Move to import section so that it will be imported just once

Reply:
Same.